appveyor-nightly.yml

```diff
@@ -0,0 +1,93 @@
+image: Visual Studio 2019
+
+version: '{build}'
```

Comment:
This is a copy of our old appveyor.yml. Only one creating artefact. It will run every night